packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -88,7 +88,7 @@ function CheckoutStepHeader( {
 			className={ joinClasses( [ className, 'checkout-step__header' ] ) }
 		>
 			<Stepper isComplete={ isComplete } isActive={ isActive }>
-				{ stepNumber || '•' }
+				{ stepNumber || null }
```

Comment:
Changed this to Null because otherwise the screen reader reads "BULLET, You're all set to checkout"